[docs] Missing build dependency msgfmt/gettext

**Operating System**
Fedora 37 (38 Beta)

**Obersavtion**
When building the extension from source as [described here](https://github.com/micheleg/dash-to-dock#installation-from-source), I get the following error message:

```
msgfmt -c po/ar.po -o po/ar.mo
make: msgfmt: No such file or directory
make: *** [Makefile:87: po/ar.mo] Error 127
```

**Solution**
Installing the `gettext` package from the distro maintainer's repository.

This dependency should be mentioned in the documentation.

--- a/README.md
+++ b/README.md
@@ -35,6 +35,15 @@ git clone https://github.com/micheleg/dash-to-dock.git
 make -C dash-to-dock install
 ```
 
+If `msgfmt` is not available on your system, you will see an error message like the following:
+
+```bash
+make: msgfmt: No such file or directory
+```
+
+In this case install the `gettext` package from your distribution's repository.
+
+
 ## Bug Reporting
 
 Bugs should be reported to the Github bug tracker [https://github.com/micheleg/dash-to-dock/issues](https://github.com/micheleg/dash-to-dock/issues).

